code/style.css

```
@@ -1,4 +1,193 @@
+
 html {
-  background: #0025ff;
-  color: #fff;
+  background: whitesmoke;
+  font-family: 'Arial Narrow' , sans-serif;
+}
+
+/*header*/
+header {
+  display: flex;
+  padding-bottom: 1%;
+}
+header img {
+  height: 8vw;
+  padding: 1% 2% 0 0;
+}
+.site-name {
+  align-self: center;
+  font-size: 3vw;
+}
+nav {
+  display: none;
+ }
+.hamburger {
+  display: flex;
+  width: 100%;
+  align-items: center;
+  justify-content: flex-end;
+  padding: 1% 1% 0 0;
+}
+.site-name:hover {
+color: #D3DEDC;
+}
+
+/*big news section*/
+.bignews-container {
+  display: box;
+  display: flex;
+  box-align: center;
+  align-items: center;
+  box-pack: center;
+  justify-content: center;
+  text-align: center;
+  margin-bottom: 3vw;
+}
+.bignews-image {
+  position: relative;
+  width: 100%;
+}
+.bignews-text {
+  color: #FFBD35;
+  font-weight: 1500;
+  position: absolute;
+  text-shadow: 1vw 1vw 2vw black;
+  -webkit-text-stroke: 0.05vw black;
+}
+.bignews-text h1 {
+  font-size: 8vw;
+  margin-bottom: 0;
+}
+.bignews-text p {
+  font-size: 2vw;
+  padding: 3vw;
+  /* text-shadow: 1vw 1vw 2vw black; */
+}
+.bignews-image:hover {
+  -webkit-transform: scaleX(-1);
+  transform: scaleX(-1);
+}
+
+/*small news section*/
+.smallnews-container {
+  background-color: #D3DEDC;
+  padding: 5vw 0 5vw 0;
+}
+.smallnews-items {
+  display: grid;
+  grid-template-columns: 1fr;
+  justify-items: center;
+  font-size: 1.5vw;
+  gap: 4vw;
+}
+.smallnews-container img {
+  height: 55vw;
+  width: 55vw;
+}
+.smallnews-items h1 {
+   font-weight: 600;
+   padding: 0 2vw 0 2vw;
+   text-align: center;
+}
+ .scandic {
```

With the images, if you want to you can add them all in the same line and style them at the same time. It would look something like this:
```suggestion
 .scandic, .sixt, .leisure, .virgin, .mandarin, .hertz, .greece, .ethiad {
   box-shadow: 2vw 1vw 1vw black;
   background-color: whitesmoke;
 }
```